docs/tempo/website/getting-started/tempo-in-grafana.md

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
i like this doc, but worry it will get stale fast b/c of how many feature flags are listed here and temporary features are listed here. i think we should make an effort as a team to remember to come back to this as things change.

also, thoughts on just not documenting log search? it's barely documented anywhere and probably basically unused. will it just add confusion to note it here?